cmd/atlas/main.go

```diff
@@ -20,10 +20,6 @@ func main() {
 	ctx, _ := signal.NotifyContext(context.Background(), os.Interrupt, os.Kill)
 	atlascmd.Root.SetOut(os.Stdout)
 	err := atlascmd.Root.ExecuteContext(ctx)
-	// Print error from command
-	if err != nil {
-		atlascmd.Root.PrintErrln("Error:", err)
-	}
```

Comment:
Can you elaborate?

Reply:
Cobra prints the error already by itself. No need to do it twice.